.editorconfig

```diff
@@ -3,8 +3,14 @@ root = true
 [*]
 indent_style = tab
 charset = utf-8
+end_of_line = lf
 trim_trailing_whitespace = true
 insert_final_newline = true
 
 [*.md]
 trim_trailing_whitespace = false
+
+[*.js]
+indent_style = space
+indent_size = 2
+
```

Comment:
I found the original to be irritating when it came to formatting. Perhaps I don't understand how the .editorconfig should work, so we should discuss and get on the same page.